CryptoScan.Web/Main/Models/Models.cs

```diff
@@ -1,7 +1,7 @@
 ﻿namespace CryptoScan.Web.Main.Models;
 
 public record Subscription(
-  string Email, 
+  string UserId, 
```

Comment:
So is there any way to still get the user email? I'm guessing it should be somehow possible to define scope="email" for the google auth and it should return the email as well.

I can work on that today

Reply:
I can check, but at the first look at the library I couldn't find it